src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestEngine.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AWS.Deploy.Common.IO;
+using Newtonsoft.Json;
+
+namespace AWS.Deploy.Common.DeploymentManifest
+{
+    public interface IDeploymentManifestEngine
+    {
+        string? TargetApplicationFullPath { get; set; }
```

Comment:
This shouldn't be here (Same comment as below)

Reply:
Fixed